# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/DocAccessible.cpp
# Commit: b92893ebb007
# Full Hash: b92893ebb007f76bc8b7a9c070e2938a0598a381
# Author: James Teh <jteh@mozilla.com>
# Date: 2021-10-08 09:48:33
# Regressor Bug: 1734322
# File Overlap Count: 1
# Description:
#   Bug 1734322: Cash DOM node id in the parent process a11y cache. r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D127757
# ==============================================================================

diff -r 92bfb5c41eb7 -r b92893ebb007 accessible/generic/DocAccessible.cpp
--- a/accessible/generic/DocAccessible.cpp	Fri Oct 08 00:52:39 2021 +0000
+++ b/accessible/generic/DocAccessible.cpp	Fri Oct 08 01:57:46 2021 +0000
@@ -769,6 +769,7 @@
     dom::Element* elm = accessible->Elm();
     RelocateARIAOwnedIfNeeded(elm);
     ARIAActiveDescendantIDMaybeMoved(elm);
+    accessible->SendCache(CacheDomain::DOMNodeID, CacheUpdateType::Update);
   }
 
   // The activedescendant universal property redirects accessible focus events